class BaseError(Exception):
    pass


class DataNotFoundError(BaseError):
    pass
